conda-reqs/chipyard.yaml

```diff
@@ -46,10 +46,10 @@ dependencies:
     - findutils
     - rsync
     - psutil
-    - doit=0.35.0
+    - doit
```

Comment:
This needs to be greater than at minimum. v34... doesn't work with the most recent marshal

Reply:
I'll add >= 0.35 here